eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -88,7 +89,7 @@ trait Eclair {
 
   def receive(description: String, amount_opt: Option[MilliSatoshi], expire_opt: Option[Long], fallbackAddress_opt: Option[String], paymentPreimage_opt: Option[ByteVector32])(implicit timeout: Timeout): Future[PaymentRequest]
 
-  def newAddress(): Future[String]
+  def newAddress()(implicit timeout: Timeout): Future[String]
```

Comment:
This is currently useless because the method is implemented by a call to `Eclair.getReceiveAddress` which doesn't take support timeouts. Other methods are implemented with `asks` which will understand the implicit parameter.

Reply:
Gotcha, I'll remove it then.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1461/commits/80b23340a99b886b6df1df79551b4c2b33ab093a